conf/conf.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package conf
+
+import (
+	"embed"
+	"io/fs"
+	"strings"
+)
+
+//go:embed auth.d gitignore label license locale readme app.ini
+var embedFS embed.FS
+
+// trimPath maps name to the trim prefix "conf/" path.
+func trimPath(name string) string {
+	if name == "conf/" || name == "conf" {
+		return "."
+	}
+	return strings.TrimPrefix(name, "conf/")
+}
+
+// MustAsset is like Asset but panics when Asset would return an error.
+// It simplifies safe initialization of global variables.
+func MustAsset(name string) []byte {
```

Comment:
Any chance we don't need this wrappers anymore? I think `embed.FS` can be easily [converted to `http.FileSystem` using `http.FS`](https://sourcegraph.com/search?q=context:global+http.FS%28+lang:Go+&patternType=literal).

Reply:
this function is used in internal/conf package in  config module just to process configure logic, not need to public outside use `http.FileSystem`. Keep this methed just make the source code have a smaller change.

Reply:
OK makes sense!

How much changes does it take for making `internal/conf` being able to directly consume a `embed.FS`?